launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/servlets/LoginServlet.java

```diff
@@ -37,16 +51,43 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
       String logoutUrl = userService.createLogoutURL(urlToRedirectTo);
-
       String loginMessage = "<p>Logged in as " + userEmail + ". <a href=\"" + logoutUrl + "\">Logout</a>.</p>";
-      LoginStatus loginStatus = new LoginStatus(true, loginMessage);
+
+      Query query =
+        new Query(LoginStatus.USER_FEED_KEY).addSort(LoginStatus.TIMESTAMP_KEY, SortDirection.DESCENDING);
```

Comment:
can you make a query that only returns feeds for a given user email? otherwise you'll be looping through all items in the entire table

Reply:
done.